src/mpid/ch4/netmod/ofi/ofi_send.h

```diff
@@ -340,37 +334,16 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_NM_mpi_send(MPIDI_OFI_SENDPARAMS)
     int mpi_errno;
     MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_NM_MPI_SEND);
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_NM_MPI_SEND);
-    mpi_errno = MPIDI_OFI_send(MPIDI_OFI_SENDARGS, 1, 0ULL);
-    MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_MPIDI_NM_MPI_SEND);
-    return mpi_errno;
-}
 
-#undef FUNCNAME
-#define FUNCNAME MPIDI_NM_rsend
-#undef FCNAME
-#define FCNAME MPL_QUOTE(FUNCNAME)
-MPL_STATIC_INLINE_PREFIX int MPIDI_NM_rsend(MPIDI_OFI_SENDPARAMS)
-{
-    int mpi_errno;
-    MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_NM_RSEND);
-    MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_NM_RSEND);
-    mpi_errno = MPIDI_OFI_send(MPIDI_OFI_SENDARGS, 1, 0ULL);
-    MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_MPIDI_NM_RSEND);
-    return mpi_errno;
-}
+    if (MPIDI_OFI_ENABLE_TAGGED) {
```

Comment:
Ugh. You're right. This is what I get for late night coding. I'll swap it around.